examples/audio_player_interaction/macos/Podfile

```diff
@@ -0,0 +1,43 @@
+platform :osx, '10.14'
```

Comment:
so it is compatible with MacOS 10.14?

Reply:
The plugin can be built on macOS 10.13 and above but the speech recognition code only works on macOS 10.15 and above.

When I generated the macOS project the pod file automatically set the platform as 10.14

Reply:
so it's best be at 10.15?